test/integration/specs/kbaseSampleSetViewer_test.js

```diff
@@ -0,0 +1,79 @@
+/*global describe, it, browser, expect, $, afterEach, beforeEach*/
+/* eslint {strict: ['error', 'global']} */
+'use strict';
+
+const {login, openNarrative} = require('../wdioUtils.js');
+
+describe('Test kbaseNarrativeSidePublicTab', () => {
+    beforeEach(async () => {
+        await browser.setTimeout({ 'implicit': 30000 });
+        await browser.reloadSession();
+        await login();
+        await openNarrative(58574);
+    });
+
+    afterEach(async () => {
+        await browser.deleteCookies();
+    });
+
+    it('SampleSetViewer should be added when clicked in data toolbar', async () => {
+        const notebook = await $('#notebook-container');
+        await notebook.waitForExist();
+
+        const cellsBefore = await notebook.$$('.cell');
```

Comment:
what does the double `$` do?

Reply:
It works like `document.$ or element.$` but selects all matching child elements rather than just the first

Reply:
Add a comment to explain that it's equivalent to `el.querySelectorAll`, in that case.

If we run the tests using browsers other than Chrome, will the code still function correctly?

Reply:
May be better in this case to add some more documentation about the API used in integration tests (webdriverio) as this and some other functions in this and existing tests do cause some confusion, especially as we use jquery elsewhere. (I think Erik is/has done some work on such documentation, I'll check in w/ him today)

Reply:
Yeah, like @dauglyon said, the `$$` thing is a wdio-ism - https://webdriver.io/docs/api/browser/$$.html

Maybe putting something in the integration test docs that point to that API page would be helpful?

Reply:
Chrome also assigns `$$` to `querySelectorAll`, which is what came up when I googled it.